package/gluon-hoodselector/luasrc/usr/sbin/hoodselector

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/lua
+
+-- PID file to ensure the hoodselector isn't running parallel
```

Comment:
i suggest to use the method used by the autoupdater lua script:
https://github.com/freifunk-gluon/packages/blob/v2017.1.x/admin/autoupdater/files/usr/sbin/autoupdater#L344
but maybe you have a good reason not to do so